aws-storage-s3/src/androidTest/java/com/amplifyframework/storage/s3/AWSS3StorageDownloadTest.java

```diff
@@ -101,14 +101,12 @@ public static void setUpOnce() throws Exception {
         // Upload large test file
         largeFile = new RandomTempFile(LARGE_FILE_NAME, LARGE_FILE_SIZE);
         key = LARGE_FILE_NAME;
-        local = largeFile.getAbsolutePath();
```

Comment:
Can we kill the declaration on L96, now? Is `String local` still used?

Reply:
It isn't, I missed it. TIL: Analyze -> Run Inspection By Name... -> Unused Declarations!